DuckDuckGo/Tab/View/BrowserTabViewController.swift

```diff
@@ -124,13 +125,30 @@ final class BrowserTabViewController: NSViewController {
         subscribeToTabs()
         subscribeToSelectedTabViewModel()
 
+        if let webViewContainer {
+            removeChild(in: self.containerStackView, webViewContainer: webViewContainer)
+        }
+
         view.registerForDraggedTypes([.URL, .fileURL])
     }
 
+     @objc func windowDidResignActive(notification: Notification) {
+         guard let webViewContainer else { return }
+         containerStackView.arrangedSubviews.filter({ $0 != webViewContainer }).forEach {
```

Comment:
we can probably move 
```
 containerStackView.arrangedSubviews.filter({ $0 != webViewContainer }).forEach {
             containerStackView.removeArrangedSubview($0)
             $0.removeFromSuperview()
         }
``` 
in a function and re-use it in viewWillDisappear as well

Reply:
This is actually removed in the follow up PR